processor/attributesprocessor/config_test.go

```diff
@@ -53,12 +50,9 @@ func TestLoadingConfig(t *testing.T) {
 		},
 	})
 
-	p1 := cfg.Processors["attributes/update"]
+	p1 := cfg.Processors[config.NewIDWithName(typeStr, "update")]
```

Comment:
Maybe too late to change :). But would these be better as something like:

```go
typ := Type(typeStr)
typ.NewID("update")
```

So would look something like:

```go
p1 := cfg.Processors[typeStr.NewID("update")]
```